tasks/insights-client.yml

```diff
@@ -15,6 +15,30 @@
   when:
     - >-
       rhc_state | d("present") == "reconnect"
-      or "NOT" in __rhc_insights_status.stdout
+      "NOT" in __rhc_insights_status.stdout
```

Comment:
careful, the "or" here is definitely needed!

Reply:
ouch I forgot to add it back